community-build/test/scala/dotty/communitybuild/CommunityBuildTest.scala

```diff
@@ -300,6 +300,16 @@ object projects:
     sbtTestCommand = ";json-schemaJVM/compile;algebraJVM/compile;openapiJVM/compile;http4s-server/compile;http4s-client/compile;play-server/compile;play-client/compile;akka-http-server/compile;akka-http-client/compile"
   )
 
+  lazy val catsEffect2 = SbtCommunityProject(
+    project        = "cats-effect-2",
+    sbtTestCommand = "test"
+  )
+
+  lazy val catsEffect3 = SbtCommunityProject(
+    project        = "cats-effect-3",
+    sbtTestCommand = "test"
```

Comment:
Oh, this one probably needs to be `testIfRelevant` for the moment. I'm working on fixing that.